src/index.html

```diff
@@ -1,13 +1,282 @@
+  <!-- #region head-->
 <!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>Nothing</title>
+    <link rel="stylesheet" href="./styles/style.scss">
     <script src="scripts/main.js" defer></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      rel="shortcut icon"
+      href="./images/favicon.png"
+      type="image/x-icon"
+    >
+
+    <link href="https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@300;400;500;600;700&family=Space+Mono:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Landing Page</h1>
+    <header class="header page-cover">
+      <div class="menu__top">
+        <a href="#">
+          <img
+            src="images/Logo.svg"
+            alt="Nothing Logo"
+            class="header__logo"
+          >
+        </a>
+
+        <div class="header__icons">
+          <a href="#" class="icon icon--phone"></a>
+          <a href="#sidebar-menu" class="icon icon--sidebar--menu"></a>
+        </div>
+      </div>
+
+      <div class="header__bottom">
+        <h1 class="header__title">Bring joy back to the everyday</h1>
+      </div>
+    </header>
+
+    <aside class="sidebar__menu page-cover" id="sidebar-menu">
+      <div class="sidebar__menu__top menu__top">
+        <a href="#">
+          <img
+            src="images/Logo.svg"
+            alt="Nothing Logo"
+            class="sidebar__menu__logo"
+          >
+        </a>
+        <a href="#" class="icon icon--close"></a>
+      </div>
+
+      <nav class="nav">
+        <ul class="sidebar__menu__links">
+          <li class="nav__item">
+            <a href="#" class="sidebar__menu__link">Products</a>
```

Comment:
Menu items should link to the respective content in the page [Checklist #10](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md#:~:text=Ensure%20that%20all%20nav__links%20work%20smoothly)

Reply:
My page hasn't a 'storage' and 'login' section but has only a 'product' section! I just put that link ;)